README.md

```diff
@@ -85,6 +85,9 @@ act_connected_nodes_range: 192.168.0.128/25
 
 # Use older style ACT topology connections (nodes[].neighbors)
 act_use_old_connections: false
+
+# internet for veos devices
+act_internet_veos: false
```

Comment:
Thanks for the contribution! I would like this key to be named act_veos_internet_access instead.